### PR TITLE
kvm-device-plugin: Fix image reference

### PR DESCRIPTION
**Pull request by cgwalters**
We need to fully qualify this; not sure how it worked before.

### CODE REVIEW

File: clusters/build-clusters/02_cluster/kvm-device-plugin/05-daemonset.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ spec:
       serviceAccountName: kvm-device-plugin
       containers:
       - name: kvm-device-plugin
-        image: kvm-device-plugin:latest
+        image: registry.build02.ci.openshift.org/kvm-device-plugin/kvm-device-plugin:latest
```
[the remark this answers is not in the document]
Comment (hongkailiu) [reply clusters/build-clusters/02_cluster/kvm-device-plugin/05-daemonset.yaml:25]: Do you have some logs showing that kubelet pulls the image from dockerhub?

[the remark this answers is not in the document]
Comment (hongkailiu) [reply clusters/build-clusters/02_cluster/kvm-device-plugin/05-daemonset.yaml:25]: Found it myself.

```
Warning  Failed          21m (x37 over 3h6m)  kubelet            Failed to pull image "kvm-device-plugin:latest": rpc error: code = Unknown desc = reading manifest latest in docker.io/library/kvm-device-plugin: errors:
denied: requested access to the resource is denied
unauthorized: authentication required
```

**Review by hongkailiu — APPROVED** [review]
https://coreos.slack.com/archives/CBN38N3MW/p1637338857345300?thread_ts=1637331343.328700&cid=CBN38N3MW

**Comment by miabbott** [comment]
Looks like this change could be necessitated because of https://bugzilla.redhat.com/show_bug.cgi?id=2000216

**Pull request merged**